README.md

```diff
@@ -25,8 +25,8 @@ An efficient but powerful RSS aggregator with a nice and mobile-friendly design,
 API (mini) How To:
 * Into your user profile: Settings > profil
 * Setup an API password
-* Check the API is working: https://exemple.tld/rss/api/greader.php
-* Setup your client with: username: ynh user, password: the password you just setup, URL https://exemple.tld/rss/api/greader.php
+* Check the API is working: https://example.tld/rss/api/greader.php
```

Comment:
```suggestion
* Check the API is working: https://example.net/rss/api/greader.php
```

See the RFC 6761 convention for documentations
https://en.wikipedia.org/wiki/Example.net